### PR TITLE
the ability to put files into config partition of EVE

### PR DESCRIPTION
**Pull request by giggsoff**
With this PR we can create files in `eve-config-dir` folder to put into config partition of EVE during setup.

Signed-off-by: Petr Fedchenkov <giggsoff@gmail.com>

### CODE REVIEW

File: docs/eve-images.md [hunk 1]
```diff
@@ -131,3 +131,9 @@ When done, you have a live image file to be used, normally in `dist/<arch>/<file
 To launch eden using the provided image:
 
 1. `eden start --image-file=path/to/your/live-image`
+
+### Overwrite config of EVE
+
+You can add files into config partition of EVE (along with the files that are generated by EdenEden) by copying them into `conf` directory.
```
Comment (mydatascience) [inline docs/eve-images.md:137]: Let's rename it to eve-conf or something... Misleading with eden config which is in /.eden

**Review by mydatascience — CHANGES_REQUESTED** [review]

File: docs/eve-images.md (hunk 1 above)
Comment (rvs) [reply docs/eve-images.md:137]: Yeah -- big +1 to renaming

File: docs/eve-images.md [hunk 2]
```diff
@@ -131,3 +131,9 @@ When done, you have a live image file to be used, normally in `dist/<arch>/<file
...
+You can select another directory you want with `--eve-config-dir` flag of `eden setup` command. To read more about config files please see
```
Comment (rvs) [inline docs/eve-images.md:138]: in fact -- you already have a perfect name ;-)

**Review by rvs — APPROVED** [review]
LGTM (sans naming comments)

File: docs/eve-images.md (hunk 1 above)
Comment (giggsoff) [reply docs/eve-images.md]: done

File: docs/eve-images.md (hunk 2 above)
Comment (giggsoff) [reply docs/eve-images.md:138]: And now I use it)

**Pull request merged**